README.md

```diff
@@ -67,6 +70,8 @@ jobs:
     - uses: actions/checkout@v2
 
     - name: Cache Primes
+      env:
+        CACHE_SKIP_SAVE: true
```

Comment:
Given that this is the default example and skipping cache save is a special case, I wouldn't include this here.  Perhaps move this into the Examples.md file and we can add some specific examples of where this is useful.

Reply:
Something we can also highlight in the example is how users can either set this on the cache step as shown here or using `echo "CACHE_SKIP_SAVE=true" >> $GITHUB_ENV`.

Reply:
I think the changes you've suggested on the readme file capture it well enough

Reply:
@roni-frantchi & @dhadka, would be great if this PR could be merged to reduce the workflow runtime when a cache-update isn't required. Seems like only a small adjustment would be needed?
Not sure if [this issue](https://github.com/actions/cache/issues/597) could be addressed in this PR too: skipping the post-step altogether when there was a cache-hit.

Reply:
> Not sure if this issue could be addressed in this PR too

I'm not sure that it can either...  
Seeing so many are expecting it, maybe we should explore doing it on another PR?..  

> Seems like only a small adjustment would be needed?

Sorry @rene-leanix it's been a while - what adjustment would that be?..

Reply:
@roni-frantchi, I was just referring to @dhadka's comments above in this thread: [here](https://github.com/actions/cache/pull/489/files#r616281599) and [here](https://github.com/actions/cache/pull/489/files#r616282355).